src/test/scala/org/apache/pekko/persistence/dynamodb/journal/DynamoDBUtils.scala

```diff
@@ -29,15 +29,15 @@ trait DynamoDBUtils extends JournalSettingsProvider with DynamoProvider {
   val system: ActorSystem
   import system.dispatcher
 
-  override lazy val journalSettings = {
+  override val journalSettings = {
```

Comment:
class has `import journalSettings._` and in scala 3, this can't be done if the val is lazy